### PR TITLE
make the user configurable a systemd timer runs as

### PR DESCRIPTION
**Pull request by zagy**

### CODE REVIEW

File: src/batou_ext/cron.py [hunk 1]
```diff
@@ -128,13 +128,19 @@ class SystemdTimer(batou.component.Component):
 
     """
 
-    _required_params_ = {"command": "/bin/true", "onCalendar": "*/4 * * * * "}
+    _required_params_ = {"command": "/bin/true", "onCalendar": "02:00:00"}
```
Comment (frlan) [inline src/batou_ext/cron.py:131]: Why this change?

Comment (zagy) [reply src/batou_ext/cron.py:131]: SystemD doesn't use cron syntax. Feel free to use a different example but the old one wouldn't even work.
